packages/mui-material/src/Grid2/Grid2.test.js

```diff
@@ -20,4 +22,31 @@ describe('<Grid2 />', () => {
     testVariantProps: { container: true, spacing: 5 },
     skip: ['componentsProp', 'classesRoot'],
   }));
+
+  it('should render with the container class', () => {
+    const { getByTestId } = render(<Grid2 data-testid="grid" container />);
+    expect(getByTestId('grid')).to.have.class(classes.container);
```

Comment:
Let's use `screen` to access queries. See https://github.com/testing-library/eslint-plugin-testing-library/blob/main/docs/rules/prefer-screen-queries.md and https://github.com/mui/material-ui/pull/42968.